### ISSUE TITLE
MediaProxy: Default expiry time (1 hour) is too short

### ISSUE THREAD

**Issue by funderscore1** [matrix-org/matrix-appservice-irc#1818]
The current default expiry time is 1 hour https://github.com/matrix-org/matrix-appservice-irc/blob/develop/config.sample.yaml#L609 which is too short.

This *will* lead to the following situation:

* Matrix user sends a long message/file
* IRC user looks at the message a few hours/days later, finds out that the link is now broken

which is what happened to me.

The default should be increased to e.g. a week or month, which is IMO saner than just an hour. IRC users (myself included) don't look at their IRC client 24/7 so this kind of situation *will* happen very frequently with such a small timeframe.

**Comment by Half-Shot** [comment]
Yeah, I tend to agree here. This came up in review out-of-band but never made it into the defaults file. I suspect something like splitting the difference and saying 14 days aught to be enough time.

**Comment by funderscore1** [comment]
As long as it's more than a week, I'm ok.

I was told the OFTC bridge's expire time was set to 8 hours - an improvement, but still too short.

**Comment by progval** [comment]
Does it really have to have an expiry at all? Given how often the bridge pastebins messages, this makes IRC chat logs unusable in some channels. And because Matrix rooms often have a `"joined"` history visibility, it means you can't even register on Matrix and join the room afterward to get the missing messages.

**Comment by funderscore1** [comment]
ping?

**Comment by whitequark** [comment]
> Does it really have to have an expiry at all? Given how often the bridge pastebins messages, this makes IRC chat logs unusable in some channels

Yes, the current default value is not well-considered from the point of view of someone actually using the bridge. ~~What's worse is that I tried setting `ttlSeconds` to 0, which sometimes works and sometimes doesn't (and I'm [not the only one hitting this issue](https://github.com/matrix-org/matrix-appservice-irc/issues/1825#issuecomment-2365191199)).~~ Upon investigation it was a misunderstanding between me and NixOS. The default is still really problematic though.
